WhaleSpotting/Controllers/LikesController.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Models.Response;
+using WhaleSpotting.Models.Request;
+using WhaleSpotting.Services;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
+[Route("likes")]
+public class LikesController : ControllerBase
+{
+    private readonly ILikeService _likesService;
+
+    public LikesController(ILikeService likesService)
+    {
+        _likesService = likesService;
+    }
+    [HttpPost("createLike")]
+    public IActionResult CreateLike([FromBody] LikeRequest newLike)
```

Comment:
We can use something like this `[FromHeader(Name = "Authorization")] string authHeader` to get the authHeader as an additional parameter
```suggestion
    public IActionResult CreateLike([FromBody] LikeRequest newLike, [FromHeader(Name = "Authorization")] string authHeader)
```

Reply:
Then we can get the user ID from the database based on the username in the AuthHeader